css/properties/scrollbar-gutter.json

```
@@ -0,0 +1,66 @@
+{
+  "css": {
+    "properties": {
+      "scrollbar-width": {
```

Looks like a little copy-and-paste mishap here:

```suggestion
      "scrollbar-gutter": {
```